packages/roosterjs-editor-plugins/lib/BlockQuote/BlockQuote.ts

```diff
@@ -0,0 +1,75 @@
+import { getBlockQuoteElement, getNodeAtCursor } from 'roosterjs-editor-api';
+import { Editor, EditorPlugin } from 'roosterjs-editor-core';
+import {
+    PluginDomEvent,
+    PluginEvent,
+    PluginEventType,
+    ContentScope,
+    ContentPosition,
+} from 'roosterjs-editor-types';
+
+const KEY_ENTER = 13;
+
+// An editor plugin to handle blockquote event
+export default class BlockQuote implements EditorPlugin {
+    private editor: Editor;
+
+    public initialize(editor: Editor): void {
+        this.editor = editor;
+    }
+
+    public dispose(): void {
+        this.editor = null;
+    }
+
+    // Handle the event
+    public onPluginEvent(event: PluginEvent): void {
+        let nodeAtCursor = getNodeAtCursor(this.editor);
+        let [isBlockQuoteEvent, blockQuoteElement] = this.isBlockQuoteEvent(event, [KEY_ENTER], nodeAtCursor);
+
+        if (isBlockQuoteEvent) {
+            let contentTraverser = this.editor.getContentTraverser(ContentScope.Selection);
+            let blockElement = contentTraverser.currentBlockElement;
+
+            // If current block is empty, delete the corresponding node and insert a br node after current blockquote
+            let content = blockElement.getTextContent().replace(/\u200B/g, '');
```

Comment:
what does "/\u200B/g" mean?

Reply:
This is for removing zero white spaces. Added comment.